test/util.ts

```diff
@@ -1386,8 +1360,10 @@ describe('common/util', () => {
       };
       const decoratedQs = {};
 
-      utilOverrides.replaceProjectIdToken = (qs: {}, projectId_: string) => {
```

Comment:
This didn't actually have any effect, since we modularized out the projectify stuff.